src/components/Nav/Nav.js

```diff
@@ -1,19 +1,20 @@
 import './Nav.css';
-import { NavLink, Link, Navigate, useNavigate } from 'react-router-dom';
+import { NavLink, Link, useLocation, useNavigate } from 'react-router-dom';
```

Comment:
great use/update of 'useLocation' Hook here - allowing the navigation elements to change based on the user's authentication status and the current URL path.